Sources/Components/HappinessRating/HappinessRatingView.swift

```diff
@@ -163,23 +163,31 @@ private class RatingImageView: UIImageView {
         setup()
     }
 
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
     func setup() {
         if delegate != nil {
             guard let text = delegate?.ratingImageViewText(for: rating) else { return }
-            let textLabel = rating.textLabel(text: text)
-            addSubview(textLabel)
+            let label = textLabel(text: text)
+            addSubview(label)
 
             let paddingToFitTwoLongWords: CGFloat = 3 // e.g: Veldig irriterende
 
             NSLayoutConstraint.activate([
-                textLabel.topAnchor.constraint(equalTo: bottomAnchor, constant: .smallSpacing),
-                textLabel.leadingAnchor.constraint(equalTo: leadingAnchor),
-                textLabel.trailingAnchor.constraint(equalTo: trailingAnchor, constant: paddingToFitTwoLongWords)
+                label.topAnchor.constraint(equalTo: bottomAnchor, constant: .smallSpacing),
+                label.leadingAnchor.constraint(equalTo: leadingAnchor),
+                label.trailingAnchor.constraint(equalTo: trailingAnchor, constant: paddingToFitTwoLongWords)
             ])
         }
     }
 
-    required init?(coder aDecoder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
+    func textLabel(text: String) -> Label {
```

Comment:
This could be private 🕵

Reply:
The class is `private` I assume that the methods should also be `private` unless explicitly set to `public`?

Reply:
The class will be private within the scope of the file, but the methods will be internal, aka. accessible within the file scope.

I forgot that the class was private, so then it's only nitpicking 🙃